### PR TITLE
feat(navigation-secondary): add roving tabindex controller

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Add roving tabindex controller to navigation-secondary top level nav slotted links.

- [x] Add changeset

## Testing Instructions

1. View Deploy preview

### CODE REVIEW

File: elements/rh-navigation-secondary/rh-navigation-secondary.ts [hunk 1]
```diff
@@ -234,6 +243,20 @@ export class RhNavigationSecondary extends LitElement {
...
+    if (this._nav) {
+      this._nav.forEach(nav => {
+        this.#navItems = Array.from(
+          nav.querySelectorAll(':is(rh-navigation-secondary-dropdown, rh-secondary-nav-dropdown) > a, [slot="nav"] > li > a')
+        );
+      });
+      if (!this.#navItems) {
+        return;
+      }
+      this.#tabindex.initItems(this.#navItems);
```
Comment (bennypowers) [inline elements/rh-navigation-secondary/rh-navigation-secondary.ts:256]:
```suggestion
    this.#tabindex.initItems(this.#navItems ?? []);
```

six of one... 
this comment is purely aesthetic, feel free to pass over it

**Review by bennypowers — COMMENTED** [review]
L...pretty?GTM

File: elements/rh-navigation-secondary/rh-navigation-secondary.ts [hunk 2]
```diff
@@ -67,6 +69,8 @@ export class RhNavigationSecondary extends LitElement {
   @colorContextProvider()
   @property({ reflect: true, attribute: 'color-palette' }) colorPalette: NavPalette = 'lighter';
 
+  @queryAssignedElements({ slot: 'nav' }) private _nav?: HTMLElement[];
```
[the remark this answers is not in the document]
Comment (zeroedin) [reply elements/rh-navigation-secondary/rh-navigation-secondary.ts:72]: yeah the native decorators will be nice to no longer have to do this for private decorated fields

**Review by bennypowers — APPROVED** [review]
Code looks good to me. @nikkimk would you please apply your usability expertise here?

**Review by brianferry — CHANGES_REQUESTED** [review]
Questions about RTI functionality - blocker

**Comment by zeroedin** [comment]
> Questions about RTI functionality - blocker

from @nikkimk in chat:
esc -> closes and sets focus on the last active tab
tab -> closes and sets focus on the next tab
shift+tab -> closes and sets focus on previous tab

**Review by brianferry — APPROVED** [review]
Keyboard navigation in the demo looks good to me, great job!

**Review by nikkimk — APPROVED** [review]
lgtm

**Pull request merged**